src/lib.rs

```diff
@@ -257,12 +256,6 @@ extern crate wasm_bindgen_test;
 #[cfg(all(target_arch = "wasm32", test))]
 use wasm_bindgen_test::*;
 
-#[allow(unused_imports)]
-use core::{
-    /* arch (handled above), */ cmp, default, f32, f64, fmt, hash, hint,
-    intrinsics, iter, marker, mem, ops, ptr, slice,
-};
```

Comment:
So the other comments mention inclusion in `core` or `std`, but this was the part that makes it easy. You can check `rust-lang-nursery/stdsimd` for how that works, there are top level directories like `coresimd`, and then there are shims in `crates/coresimd` to handle inclusion in std.